src/getting-started/registering-an-account.md

```diff
@@ -1,30 +1,91 @@
+3. When ready, select **Next**.
+4. On the **Specify company details** screen that appears, specify the following details:
+   - **Company details**:
+     - **Company name** — full legal name of the business that the account is associated with.
+     - **Operating country** — the country of operation of the business that the account is associated with.\
+     Selected from a dropdown menu; only one country can be selected.
+     - **Membership type** — the type of the business that the account is associated with.\
+     Can be one of the following:
+       - **Telco** — telecommunication company that provides a number of various telecommunications services (e.g., telephony, internet access, etc.) to their end customers.\
+       If **Telco** is selected, then its **Number of subscribers** must also be specified in the dropdown menu that appears.
+       - **Vendor** — company that provides telco companies with all-level facilities and support, including installation and maintenance of hardware.
+   - **Personal details**:
+     - **First name** — your full first name.
+     - **Last name** — your full last name.
+     - **Your role in company** — the position that you occupy in the business that the account is associated with.
+     - **Phone number** — your contact phone number, starting with your country's dial-in code (e.g., `+81` for Japan).
+5. When ready, select **Next**.
+6. On the **Account credentials** screen that appears, specify the following details:
+   - **Your email** — the email address that is registered under your business' domain and is only accessible by you.
+   - **Password** — a unique combination of symbols that will be used to log in to your account via the Web App.
+   - **Repeat password** — same as **Password**; must be entered to ensure that no typos were entered the first time.
+
+   ::: tip Note
+
+   Your password must adhere to the following rules:
+     - Must be at least six characters in length;
+     - Must contain at least one lower case letter (e.g., `a`-`z`);
+     - Must contain at least one upper case letter (e.g., `A`-`Z`);
+     - Must contain at least one numeral character (e.g., `0`-`1`);
+     - Must contain at least one special character (e.g., `?`, `#`, `(`, `[`, etc.).
+
+   **Example**: `Xfj&Hv7k`
+
+   :::
+
+   ::: warning
+
+   Don't use the same password that you use for authorization on any other online application and/or service.
+
+   For a deeper understanding of the password security principles, you may want to read the following topic of the Iroha 2 documentation: [Iroha 2 > Security > Password Security](https://docs.iroha.tech/guide/security/password-security.html).
+
+   :::
+
+7. Review the [Terms & Conditions](https://github.com/fraud-intelligence-limited/fil-legal/blob/main/Fraud%20Intelligence%20Blockchain%20Terms%20of%20Use%20v6.0.md), then select the corresponding **I agree with the Terms and Conditions** checkbox.
```

Comment:
the version of that might be updated, let's better refer to the main branch of the repo only